parts/k8s/windowslogscleanup.ps1

```diff
@@ -0,0 +1,25 @@
+<#
+.DESCRIPTION
+    This script cleans old rotated logs for various kubernetes components.
+#>
+
+$global:LogPath = "c:\k\windowslogscleanup.log"
+
+filter Timestamp { "$(Get-Date -Format o): $_" }
+
+function Write-Log ($message) {
+    $message | Timestamp | Tee-Object -FilePath $global:LogPath -Append
+}
+
+Write-Log "Entering windowslogscleanup.ps1"
+
+$logFilePrefixes = @("kubelet", "kubelet.err", "kubeproxy", "kubeproxy.err")
+
+foreach ($logFilePrefix in $logFilePrefixes) {
+    $oldLogs = [IO.Directory]::GetFiles("c:\temp\kubelogs\logs", "$($logFilePrefix)-*.log")
```

Comment:
is `c:\temp\kubelogs\logs` right? shouldn't it just be c:\k ?

Reply:
Whoops, those were paths I was using the test the powershell and scheduled task locally.